features/yield/FarmingCalculator.tsx

```diff
@@ -314,7 +311,207 @@ const FarmingCalculator = () => {
       </Box>
     );
   } else {
-    return <LiquidityMiningV2 />;
+    return (
+      <span>
+        <Typography variant="h5">UMA Liquidity Mining</Typography>
+
+        <br></br>
+        <Typography>
+          During the liquidity mining program UMA and potentially other token
+          rewards will be paid out to LP providers in certain yield token
+          balancer pools. Rewards are calculated as a pro-rata contribution to
+          the liquidity pool. To learn more about the liquidity mining program
+          see UMA Medium post{" "}
+          <a
+            href="https://medium.com/uma-project/liquidity-mining-on-uma-is-now-live-5f6cb0bd53ee"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            here
+          </a>
+          .
+        </Typography>
+        {timeUntilRoll && timeUntilRoll > 0 && (
+          <>
+            <br></br>
+            <br></br>
+            <Typography>
+              <strong>Liquidity mining rewards during the roll:</strong> Before{" "}
```

Comment:
Do you think its worth it adding any details about dev mining here for the user to better understand their rewards?